.github/workflows/pre-build.yaml

```diff
@@ -1,16 +1,16 @@
 # this workflow will try to lint and build a node.js application
 #
-# this is useful for stages that you require to make sure everything is working
+# this is useful for stages that you require making sure everything is working
 # properly before creating a container image to be pushed on the cloud
 #
-name: pre_build
+name: openverse/validate
 
 on:
   pull_request:
-  push:
-    branches:
-      - 'main'
-      - 'ci/*' # branches that follows the pattern ci/* can access this workflow too
+#  push:
+#    branches:
+#      - 'main'
+#      - 'ci/*' # branches that follows the pattern ci/* can access this workflow too
```

Comment:
Is this disabled for any particular reason?

Reply:
yes, in my opinion, if we are running this script on `pull_request` event (opened, closed, updated, reopened, etc), it doesn't make much sense to enable it when the `main` branch es pushed.

In other words, when we merge a Pull Request into `main`, we are triggering this workflow twice.

Reply:
Got it, should we just delete the file then?

Reply:
No, with the lines already commented is sufficient. I can remove the lines if you want

Reply:
Oh sorry I forgot this is the _same action_ as the one that runs on PRs, not a different one.